src/main/java/org/team1540/robot2024/subsystems/vision/gamepiece/NoteVisionIO.java

```diff
@@ -0,0 +1,21 @@
+package org.team1540.robot2024.subsystems.vision.gamepiece;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface NoteVisionIO {
+    @AutoLog
+    class NoteVisionIOInputs {
+        public boolean hasDetection = false;
+        public double lastDetectionTimestampSecs = 0.0;
+        public double targetPitchRads = 0.0;
+        public double targetYawRads = 0.0;
+        public double targetAreaRads = 0.0;
```

Comment:
I find it hard to believe that this is in radians

Reply:
:skull: